kernel/build-kernel.sh

```diff
@@ -272,10 +272,12 @@ get_config_and_patches() {
 		info "Clone config and patches"
 		patches_path="${default_patches_dir}"
 		if [ ! -d "${patches_path}" ]; then
-			tag="${kata_version:-$NEW_VERSION}"
+			tag="${kata_version}"
 			git clone "https://${patches_repo}.git" "${patches_repo_dir}"
 			pushd "${patches_repo_dir}" >> /dev/null
-			git checkout $tag
+			if [ ! -z $tag ] ; then
```

Comment:
this could be `-n`, but fine if want to merge this way.

Reply:
fixed